src/commands/deploy.ts

```diff
@@ -3,14 +3,13 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// tslint:disable-next-line:no-require-imports
-import WebSiteManagementClient = require('azure-arm-website');
 import { StringDictionary } from 'azure-arm-website/lib/models';
 import * as fse from 'fs-extra';
 import * as path from 'path';
 import * as vscode from 'vscode';
 import { MessageItem } from 'vscode';
-import { SiteWrapper } from 'vscode-azureappservice';
+import { SiteClient } from 'vscode-azureappservice';
+import * as appservice from 'vscode-azureappservice';
```

Comment:
Import from the same package twice?

Reply:
I fixed a few cases, but I actually do this on purpose sometimes. In most cases I would prefer to do this:
`import { deploy } from 'vscode-azureappservice';`

But in this case I couldn't do that because I already have a `deploy` function defined in this file. So I have to do `import * as appservice from 'vscode-azureappservice';` and call `appservice.deploy`. But I only use that syntax if absolutely necessary and leave the rest as I would prefer it above.

So yeah that's my reasoning, but at the end of the day I don't have a strong preference on this